charts/stable/authelia/questions.yaml

```diff
@@ -912,6 +912,16 @@ questions:
                                         description: "one_factor"
                                       - value: "two_factor"
                                         description: "two_factor"
+                                - variable: consent_mode
+                                  label: "Consent Mode"
+                                  description: "Configures the consent mode. This can be set to auto (default), explicit (consent required every time) or implicit (Automatically assumes consent for every authorization, never asking the user if they wish to give consent."
+                                  schema:
+                                    type: string
+                                    default: "auto"
+                                    enum:
+                                      - value: "auto"
+                                      - value: "explicit"
+                                      - value: "implicit"
```

Comment:
You need both value and description.
description is the visible part

Reply:
Done. Meant to add them and got distracted by food.